### ISSUE TITLE
darwin-aarch64 build?

### ISSUE THREAD

**Issue by RankoR** [ValveSoftware/Dota-2#1908]
Is the Apple Silicon build expected? For now, on Macbook Pro M1 dota runs in 50-60 fps on 4K with minimal graphical settings in x86 emulation mode, but it would be nice to have a native build.

**Comment by Turmfalke2** [comment]
Steam itself is only available for x86/amd64, until that changes dota2 is unlikely to get an aarch64 build.

related ticket:
https://github.com/ValveSoftware/steam-for-linux/issues/4061

**Comment by vrodic** [comment]
wasn't M1 build available previously? how to use it now?

**Comment by edchelstephens** [comment]
Is there a progress on this issue?

**Comment by TangMonk** [comment]
M1 support please

**Comment by levihuayuzhang** [comment]
> Steam itself is only available for x86/amd64, until that changes dota2 is unlikely to get an aarch64 build.
> 
> related ticket: [ValveSoftware/steam-for-linux#4061](https://github.com/ValveSoftware/steam-for-linux/issues/4061)


How about the new steamwork? Seems like ARM works?

https://www.youtube.com/watch?v=X5uCc1BJGX0
and
https://store.steampowered.com/news/group/4145017/view/2984186817523157398
and here
https://www.youtube.com/watch?v=3AxkhWhSWDI

**Comment by levihuayuzhang** [comment]
BTW, is the FidelityFX™ Super Resolution actually works on Apple silicons?

Cause on AMD's official [site](https://www.amd.com/en/technologies/fidelityfx-super-resolution), the fsr seems not works natively.

**Comment by heng4fun** [comment]
Any progress?

**Comment by novasvilla** [comment]
Any progress? Or at least any hope? Imagine how nice can be a good native apple Silicon/ metal 3 with up scaling  :)

**Comment by liho00** [comment]
Need dota 2 arm64 native support, just upgraded to m2 max, but my macbook become hot after start dota 2 for 15 mins... I worry it hurt the laptop hence decide stop playing dota until it got supported...

**Comment by levihuayuzhang** [comment]
I guess the Apple Silicon users are abandoned by Valve...
How many years?
Even no plan for this?

**Comment by levihuayuzhang** [comment]
> I guess the Apple Silicon users are abandoned by Valve...
> How many years?
> Even no plan for this?

Using Github issue not means you are open source project or non-profit...
We are the users and players that may spent a lot in this game...
Try to support the community plz

**Comment by dungmv** [comment]
Should I buy a new pc or wait for native apple Silicon/metal?

**Comment by MykolasPinkevicius** [comment]
Waiting someone to build dota2 on metal. I wonder why nobody wants to at least allocate people to this cause. You make people to learn develop on apple silicone and then you can allocate them to make projects to optimise for apple silicone and get a lot of projects.

**Comment by LancerTZ** [comment]
Any Progress？ Until now, what's the key reason for blocking this? Is there anything we can do to move this forward?

**Comment by liho00** [comment]
guys just use Nvidia GeforceNow to play Dota 2 on Mac, its smooth and without lag, I tried it.
https://www.nvidia.com/geforce-now/

**Comment by RomanCRS** [comment]
Any update on this?

**Comment by LancerTZ** [comment]
> guys just use Nvidia GeforceNow to play Dota 2 on Mac, its smooth and without lag, I tried it. https://www.nvidia.com/geforce-now/

I tried Nvidia GeforceNow before. There are 3 main disadvantages:
1. During team battles, the game interface becomes blurry, and I can't even distinguish the edges of the hero's model to cast abilities.
2. The GFN server doesn't have a good connectivity with Dota 2 server in my expected region. And sometimes I just want to demo hero or test something with bots offline.
3. Nvidia GeforceNow needs additional money.

**Comment by LancerTZ** [comment]
> Any update on this?

Nope. It seems that the developers aren't paying attention to this issue.

**Comment by levihuayuzhang** [comment]
lmao Valve is all in Linux now...
As for Darwin ARM? Maybe they even haven't got an Apple Silicon Machine for developing...

**Comment by huybik** [comment]
Tradditonally Apple support their device for 8 years. The last batch of intel machine was on 2022, so 2030 will see Apple kill off X86 support. We will get Dota darwin-aarch64 build if the game hasnt went the way of the dodo by then.

**Comment by LancerTZ** [comment]
> Tradditonally Apple support their device for 8 years. The last batch of intel machine was on 2022, so 2030 will see Apple kill off X86 support. We will get Dota darwin-aarch64 build if the game hasnt went the way of the dodo by then.

Are we really going to have to wait until 2030? I really hope to play the native version of DOTA 2 on the upcoming redesigned M4 Pro Mac mini.

**Comment by RaresAndrei0** [comment]
> > Tradditonally Apple support their device for 8 years. The last batch of intel machine was on 2022, so 2030 will see Apple kill off X86 support. We will get Dota darwin-aarch64 build if the game hasnt went the way of the dodo by then.
> 
> 
> 
> Are we really going to have to wait until 2030? I really hope to play the native version of DOTA 2 on the upcoming redesigned M4 Pro Mac mini.

That feels optimistic, I think valve is going to drop support like they did with CS2 when the slightest inconvenience will rise

**Comment by thomas992** [comment]
> > Tradditonally Apple support their device for 8 years. The last batch of intel machine was on 2022, so 2030 will see Apple kill off X86 support. We will get Dota darwin-aarch64 build if the game hasnt went the way of the dodo by then.
> 
> Are we really going to have to wait until 2030? I really hope to play the native version of DOTA 2 on the upcoming redesigned M4 Pro Mac mini.

Apple has declared all their Intel builds to be obsolete. Valve needs to add proper support for Mac.

**Comment by huybik** [comment]
Ok now steam is native, it is about time for dota native.

**Comment by Boing57** [comment]
Steam now run native on Apple Silicon. It time for Dota 2. Please

**Comment by nikitapilgrim** [comment]
+

**Comment by nimseven7** [comment]
Hi! Is there any news on this matter? Having some issues with CPU usage on MacBook Pro m4 pro (sadly on macOS Tahoe).
We do understand that the game porting kit won't work as Dota 2 already uses Vulkan on macOS.
Even some articles about advancement in the subject would be appreciated.
Thank you!

**Comment by lwolive** [comment]
Dota2 is a great game and Please support Apple Silicon native . It can run smooth even on  amd 780m ,  I think the native version will run very smooth on Apple Slilicon .

**Comment by gingerale95** [comment]
Have stutter lag on my m3 16gb Macbook Pro, native DOTA 2 would 100% resolve this as i dont have this issue on other "higher demanding" native online games

**Comment by RaresAndrei0** [comment]
> Have stutter lag on my m3 16gb Macbook Pro, native DOTA 2 would 100% resolve this as i dont have this issue on other "higher demanding" native online games

moved to LoL, they just released the native version of the game :)

**Comment by vuon9** [comment]
Apple is officially phasing out Rosetta 2. It will no longer be available for general applications starting with macOS 28.

**Comment by huybik** [comment]
it's great to hear that finally we will not able to play dota 2 on mac when it transition fully out of x86 simulation, similar to how cs go 2 abandoned arm completly. and then the whole industry transitions to arm and we finally can't play valve games anymore 😢

**Comment by zuozhiw** [comment]
Please valve use latest AI models to vibe code mac support plz

**Comment by dafadhiya** [comment]
MacOS Golden Gate is coming very soon, they will not support dota anymore if Valve not porting it using GPTK and run natively on arm. please valve, i know there just small % of player using mac, but cmon

**Comment by sandeepgahlawat** [comment]
I’ve been playing Dota 2 for a decade now, and to me, nothing else even compares. Unfortunately, I have to use macOS for my day job, and I just can't justify buying a separate Windows PC just for one game. Ever since macOS Tahoe, I've been plagued with constant stuttering and lag. Now on macOS 27, it's completely unplayable.Valve, you have the resources to keep this game running beautifully across platforms. Please don't ignore your hardcore Mac players. We love this game and just want to be able to play it properly again!

**Comment by vuon9** [comment]
> I’ve been playing Dota 2 for a decade now, and to me, nothing else even compares. Unfortunately, I have to use macOS for my day job, and I just can't justify buying a separate Windows PC just for one game. Ever since macOS Tahoe, I've been plagued with constant stuttering and lag. Now on macOS 27, it's completely unplayable.Valve, you have the resources to keep this game running beautifully across platforms. Please don't ignore your hardcore Mac players. We love this game and just want to be able to play it properly again! 

 I think it still can work on macOS 27, pls reinstall Rosetta and try again. It will only be limited when macOS 28 comes
